etc/config/snowball.amazon.properties

```diff
@@ -22,5 +22,5 @@ group.snowball.supportsBinary=true
 group.snowball.supportsBinaryObject=true
 
 # \_0_/
-compiler.snowball07.exe=/opt/compiler-explorer/snowball-0.0.7/bin/snowball
-compiler.snowball07.semver=0.0.7
+compiler.snowball08.exe=/opt/compiler-explorer/snowball-0.0.8/bin/snowball
```

Comment:
As discussed in Discord, this "breaks" any URL to the old 07 version. But the broken link will notice the missing compiler and will then select the default, so... I think this will "work" though it looks like you've changed the syntax.

Reply:
The language is not really that famous... *yet* 😈.

So I think it doesn't really matter